.circleci/post-release.yaml

```diff
@@ -243,20 +243,21 @@ jobs:
             vless="${VERSION:1}"
             rcless="${vless%%-*}"
             formula_version="${rcless%.*}"
-            # branch="update_aperture_${formula_version}"
-            branch=main
+            branch="update_aperture_${VERSION}"
+            msg="Update brews for release ${VERSION}"
+
 
-            # git checkout -B "${branch}"
+            git checkout -B "${branch}"
             ./scripts/update_brews.py add-version "${formula_version}"
             ./scripts/update_brews.py delete
             ./scripts/update_brews.py update
 
             git status
             git diff
             git add .
-            git commit -m "Update brews for release ${VERSION}"
+            git commit -m "${msg}"
             git push origin "${branch}"
-            # gh pr create --title "${MSG}" --body "" --label "pr-pull"
+            gh pr create --title "${msg}" --body "" --label "pr-pull"
```

Comment:
I think I tried this and it is not triggering the release on brew so should we merge directly to main so that we don't have to manually merge this PR?

Reply:
We were missing the label earlier. And this might fix it too: https://github.com/fluxninja/homebrew-aperture/pull/20